Test/PacketType/DrdaPacketTest.cs

```diff
@@ -0,0 +1,103 @@
+﻿/*
+This file is part of PacketDotNet
+
+PacketDotNet is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+PacketDotNet is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with PacketDotNet.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/*
+ *  Copyright 2010 Chris Morgan <chmorgan@gmail.com>
+ */
+
+using System;
+using System.Text;
+using NUnit.Framework;
+using SharpPcap;
+using SharpPcap.LibPcap;
+using PacketDotNet;
+using System.IO;
+using System.Runtime.Serialization.Formatters.Binary;
+
+namespace Test.PacketType
+{
+    class DrdaPacketTest
+    {
+        [Test]
+        public void TestSelectPacket()
+        {
+            RawCapture raw;
+            int packetIndex = 0;
+            var dev = new CaptureFileReaderDevice("../../CaptureFiles/db2_select.pcap");
+            dev.Open();
+
+
+            while ((raw = dev.GetNextPacket()) != null)
+            {
+                var p = Packet.ParsePacket(raw.LinkLayerType, raw.Data).Extract(typeof(DrdaPacket)) as DrdaPacket;
+                if (p != null)
+                {
+                    foreach(var ddm in p.DrdaDDMPackets)
+                    {
+                        Console.WriteLine(ddm.ToString(StringOutputType.Colored));
+                    }
+                }
+                packetIndex++;
+            }
+            dev.Close();
+
+        }
+
+        [Test]
```

Comment:
This should go into a separate test file as it is specifically testing the conversion you added. The other point I'd make is that you are duplicating the array of values here. Imo you could drop that as its just a copy of what you have in the conversion routine and simply include the input array of values and what you expect as the output. Duplicating the array here isn't buying us much, we already have that and can use that out of the other class you added without duplicating it here.

Reply:
Thanks for your suggest, I have modify this method to do not copy array anymore.

Reply:
This method just to test Ebcdic To Ascii String Convert, no more other usage, so I feel test here is fine, but if you suggest to put it in a separate test file, I can do it too.